Extension/package.json

```diff
@@ -5092,17 +5092,17 @@
       "editor/context": [
         {
           "command": "C_Cpp.SwitchHeaderSource",
-          "when": "config.C_Cpp.intelliSenseEngine != 'Disabled' && editorLangId == 'c' || config.C_Cpp.intelliSenseEngine != 'Disabled' && editorLangId == 'cpp' || config.C_Cpp.intelliSenseEngine != 'Disabled' && editorLangId == 'cuda-cpp'",
+          "when": "config.C_Cpp.intelliSenseEngine != 'disabled' && editorLangId == 'c' || config.C_Cpp.intelliSenseEngine != 'disabled' && editorLangId == 'cpp' || config.C_Cpp.intelliSenseEngine != 'disabled' && editorLangId == 'cuda-cpp'",
```

Comment:
Is this comparison case sensitive?  If it is, then we need to check both 'Disabled' and 'disabled' for now.